src/aks-preview/azext_aks_preview/_params.py

```diff
@@ -484,6 +488,8 @@ def load_arguments(self, _):
         c.argument('mode', arg_type=get_enum_type(node_mode_types))
         c.argument('scale_down_mode', arg_type=get_enum_type(scale_down_modes))
         c.argument('max_surge', validator=validate_max_surge)
+        c.argument('enable_custom_ca_trust', action='store_true', validator=validate_enable_custom_ca_trust)
+        c.argument('disable_custom_ca_trust', options_list=['--disable-custom-ca-trust', '--dcat'], action='store_true')
```

Comment:
Actually, there's an ongoing thread discussing about whether to remove paired bool options, I'll forward the email to you.